core/arch/mangle_shared.c

```diff
@@ -188,6 +188,10 @@ prepare_for_clean_call(dcontext_t *dcontext, clean_call_info_t *cci, instrlist_t
 {
     uint dstack_offs = 0;
 
+    instr_t *start_label = INSTR_CREATE_label(dcontext);
+    instr_set_note(start_label, (void *)DR_NOTE_CALLOUT_SEQUENCE_START);
```

Comment:
The note doesn't say which sequence starts/ends.

Reply:
Maybe say "clean call" explicitly?

Reply:
What do you mean "which sequence starts/ends"?  The note indicates the start of inserted instructions that prepare to do a callout.

Reply:
Oh  I assumed "CALLOUT" to mean that the label is "calling out" something, which led me to ask "what is it calling out".